### ISSUE TITLE
AttributeError: module 'jsonschema._validators' has no attribute 'properties_draft4'

### ISSUE THREAD

**Issue by mnboos** [p1c2u/openapi-spec-validator#61]
This happens with openapi-spec-validator==0.2.5 and jsonschema==3.0.0

```
  File "/data/web/****/api/v2/models.py", line 7, in <module>
    from openapi_spec_validator import validate_spec
  File "/venv/lib/python3.6/site-packages/openapi_spec_validator/__init__.py", line 7, in <module>
    from openapi_spec_validator.factories import JSONSpecValidatorFactory
  File "/venv/lib/python3.6/site-packages/openapi_spec_validator/factories.py", line 5, in <module>
    from openapi_spec_validator.generators import (
  File "/venv/lib/python3.6/site-packages/openapi_spec_validator/generators.py", line 12, in <module>
    class SpecValidatorsGeneratorFactory:
  File "/venv/lib/python3.6/site-packages/openapi_spec_validator/generators.py", line 19, in SpecValidatorsGeneratorFactory
    'properties': _validators.properties_draft4,
AttributeError: module 'jsonschema._validators' has no attribute 'properties_draft4'
```

**Comment by niltz** [comment]
dupe of #54

**Comment by p1c2u** [comment]
openapi-spec-validator==0.2.5 requires sonschema<3 It was fixed with #59
